x/group/client/testutil/tx.go

```diff
@@ -526,10 +520,10 @@ func (s *IntegrationTestSuite) TestTxUpdateGroupMetadata() {
 			append(
 				[]string{
 					val.Address.String(),
-					"2",
+					"1",
```

Comment:
So because the test clashed, and the added test executed before the group was not the same, I am fixing it by trying to isolate them it wont stay like that